spring-cloud-starter-stream-sink-log/README.adoc

```diff
@@ -3,6 +3,20 @@
 
 The `log` sink uses the application logger to output the data for inspection.
 
+== Input
+
+=== Headers
+
+`content-type: text/plain`
+
+=== Payload
+
+A `String` representation of the logging message.
```

Comment:
I think that's not true.
Any object with any content-type can be used for the logging.
Just see all those `Log`  methods!
```
    /**
     * Logs a message with debug log level.
     *
     * @param message log this message
     */
    void debug(Object message);
```